bump-packages/action.yml

```diff
@@ -19,15 +19,25 @@ inputs:
 runs:
   using: composite
   steps:
-    - run: brew bump --open-pr --formulae ${{ inputs.formulae }}
+    - run: |
+        if [[ "${GITHUB_REPOSITORY_OWNER}" == 'Homebrew' ]]; then
+          brew bump --no-fork --open-pr --formulae ${{ inputs.formulae }}
+        else
+          brew bump --open-pr --formulae ${{ inputs.formulae }}
+        fi
+      shell: bash
       if: inputs.formulae != ''
-      shell: sh
       env:
         HOMEBREW_DEVELOPER: "1"
-        HOMEBREW_GITHUB_API_TOKEN: ${{inputs.token}}
-    - run: brew bump --open-pr --casks ${{ inputs.casks }}
-      if: inputs.casks != ''
-      shell: sh
+        HOMEBREW_GITHUB_API_TOKEN: ${{ inputs.token }}
+    - run: |
+        if [[ "${GITHUB_REPOSITORY_OWNER}" == 'Homebrew' ]]; then
+          brew bump --no-fork --open-pr --cask ${{ inputs.cask}}
+        else
+          brew bump --open-pr --cask ${{ inputs.cask }}
+        fi
+      shell: bash
+      if: inputs.cask != ''
```

Comment:
Sorry I missed that this changed: this should be `inputs.casks` not `inputs.cask`